README.md

```diff
@@ -16,6 +16,11 @@ There are no dependencies needed to run the website, everything is prepared to w
 
 # Structure
 
+Instead of writing all code in a single JavaScript file, we want you to split your code over several files.
+The structure of this project is explained in the next video
+
+[![Project Structure YouTube Video](https://i.imgur.com/hDcLYFt.png)](https://youtu.be/bysBqtSKBpQ)
```

Comment:
Really cool video, great idea starting with a single file and doing the splitting live! Thanks a lot for this